functions-methods.js

```diff
@@ -33,4 +68,22 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity(emailAddress){
+
+    const atMail= emailAddress.includes("@");
+    const indexAt= emailAddress.indexOf('@');
+    const commaMail=emailAddress.includes(",");
+    const period=emailAddress.includes('.');
+    const indexPeriod=emailAddress.lastIndexOf('.');
+    const emailLength= emailAddress.length-1
+
+    if(atMail && !commaMail && (indexPeriod>indexAt) && (indexPeriod!==emailAddress.length-1 )){
+        return true;
+    }
+    else {
+        return false;
+    }
```

Comment:
Dit had je nog iets kunnen versimpelen
```suggestion
    return atMail && !commaMail && indexPeriod>indexAt && indexPeriod!==emailAddress.length-1;
```